src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooBuildPlanService.java

```diff
@@ -172,9 +171,9 @@ private Stage createBuildStage(ProgrammingLanguage programmingLanguage, ProjectT
          */
         Collection<String> activeProfiles = Arrays.asList(env.getActiveProfiles());
 
-        // Do not run the builds in extra docker containers if the dev-profile is active
+        // TODO: maybe add a bamboo profile flag to not run the builds in extra docker containers? instead of binding this to the dev profile
```

Comment:
Not sure how to handle this? Just leave it as below or modify it?

Reply:
As the Atlassian stack on docker compose now also supports the build agent, this is now the default for all environments and no extra config flag is necessary.